src/client/editor/helpers/SpokePointLightHelper.js

```diff
@@ -0,0 +1,63 @@
+import THREE from "../three";
+
+export default class SpokePointLightHelper extends THREE.Mesh {
+  constructor(light, sphereSize, color) {
+    const geometry = new THREE.SphereBufferGeometry(sphereSize, 4, 2);
+    const material = new THREE.MeshBasicMaterial({ wireframe: true, fog: false });
+
+    super(geometry, material);
+
+    this.light = light;
+    this.light.updateMatrixWorld();
+
+    this.color = color;
+
+    this.matrix = this.light.matrixWorld;
+    this.matrixAutoUpdate = false;
+
+    const distanceGeometry = new THREE.IcosahedronGeometry(1, 2);
+    const distanceMaterial = new THREE.MeshBasicMaterial({
+      color: color,
+      fog: false,
+      wireframe: true,
+      opacity: 0.1,
+      transparent: true
+    });
```

Comment:
I'm not sure about having this visible at all times when a distance is set. Might get annoying. Ideally the range helper would only be visible while the point light is selected? Would be nice for a future version.

Reply:
Maybe all the light helpers should function this way?

Reply:
Hmm, I think you want some simple gizmo so that you can actually see that the lights exist in the scene, kinda like Unity's icon gizmo and then have them expand to the full helper representation when they are selected.

Reply:
Ok, yeah. Let's do that soon.